sql/src/influxql/mod.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2023 CeresDB Project Authors. Licensed under Apache-2.0.
+
+pub mod planner;
+mod stmt_rewriter;
+pub(crate) mod util;
+pub mod error {
+    use common_util::error::GenericError;
+    use snafu::Snafu;
+
+    #[derive(Debug, Snafu)]
+    #[snafu(visibility = "pub")]
+    pub enum Error {
```

Comment:
If error in this enum are all generic error, i think it make little to define so many fields, maybe two is enough

```
InternalError { err, msg }
InternalErrorWithTrace { err, msg }
```

Reply:
> If error in this enum are all generic error, i think it make little to define so many fields, maybe two is enough
> 
> ```
> InternalError { err, msg }
> InternalErrorWithTrace { err, msg }
> ```

It sounds good.